implementations/rust/ockam/ockam_command/src/error.rs

```diff
@@ -24,7 +24,7 @@ impl Error {
 
 impl Display for Error {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        std::fmt::Display::fmt(&self.inner, f)
+        std::fmt::Debug::fmt(&self.inner, f)
```

Comment:
This is to output the anyhow's backtrace error, instead of just the last error message.